### PR TITLE
Add locales to readme.md #398

### PR DESCRIPTION
**Pull request by lix-dev**
add translations to readme

Hacktober 2020

### CODE REVIEW

**Comment by lix-dev** [comment]
Can you add "hacktoberfest" to repository labels? In this case i will earn a point for helping open source projects :)

Best wishes from the northern of germany :)

File: README.md [hunk 1]
```diff
@@ -88,3 +88,19 @@ The patterns can be structured in roughly three different categories. Please cli
 * [EAV](More/EAV) [:notebook:](https://en.wikipedia.org/wiki/Entity%E2%80%93attribute%E2%80%93value_model)
 * [Repository](More/Repository)
 * [ServiceLocator](More/ServiceLocator) [:notebook:](http://en.wikipedia.org/wiki/Service_locator_pattern)
+
+## Localisation & Supported Languages
+
+| Code | Language      | Documentation |
+| ------------- |-------------| -----|
+| en | English | [Docs :notebook:](https://designpatternsphp.readthedocs.io/en/latest/README.html) |
+| ca | Catalan | [Docs :notebook:](https://designpatternsphp.readthedocs.io/en/latest/README.html) |
```
Comment (domnikl) [inline README.md:97]: That link is wrong: leads to 'en'

File: README.md [hunk 2]
```diff
@@ -88,3 +88,19 @@ The patterns can be structured in roughly three different categories. Please cli
...
+| de | Germany | [Docs :notebook:](https://designpatternsphp.readthedocs.io/de/latest/README.html) |
+| es | Spanish | [Docs :notebook:](https://designpatternsphp.readthedocs.io/es/latest/README.html) |
+| es_MX | Spanish-Mexican | [Docs :notebook:](https://designpatternsphp.readthedocs.io/es/latest/README.html) |
```
Comment (domnikl) [inline README.md:100]: should lead to 'es_MX' instead of 'es'

**Review by domnikl — CHANGES_REQUESTED** [review]
Thanks for the PR @lix-dev! If you change the broken links, I can merge it in.

File: README.md (hunk 1 above)
Comment (lix-dev) [reply README.md:97]: You mean the link for Catalan docs? What is the slug for catalan?

File: README.md [hunk 3]
```diff
@@ -88,3 +88,19 @@ The patterns can be structured in roughly three different categories. Please cli
...
 * [Repository](More/Repository)
 * [ServiceLocator](More/ServiceLocator) [:notebook:](http://en.wikipedia.org/wiki/Service_locator_pattern)
+
+## Localisation & Supported Languages
+
+| Code | Language      | Documentation |
+| ------------- |-------------| -----|
+| en | English | [Docs :notebook:](https://designpatternsphp.readthedocs.io/en/latest/README.html) |
+| ca | Catalan | [Docs :notebook:](https://designpatternsphp.readthedocs.io/ca/latest/README.html) |
+| de | Germany | [Docs :notebook:](https://designpatternsphp.readthedocs.io/de/latest/README.html) |
```
Comment (RedDevilus) [inline README.md:98]: Germany should be German, but that's just nitpicky.

**Pull request merged**